### ISSUE TITLE
Oban MySQL + Oban Pro

### ISSUE THREAD

**Issue by beerlington** [oban-bg/oban#1183]
Our team is interested in running Oban with MySQL and would love to help test it out. We're currently running Oban Pro 15 RC and are using the `Oban.Pro.Engines.Smart` engine. Is there an option for us to try this out on MySQL and still use the Pro engine?

**Comment by sorentwo** [comment]
Great to hear that you're interested in trying out the new MySQL engine!

The Smart engine won't work with MySQL at this point. It is built specifically for Postgres, and relies on a number of Postgres features internally. However, other parts of Pro that don't rely on the Smart engine will work (structured args, decorators, relay, etc.)

Our goal is to support all engines with Web and Pro, but we need to get MySQL tested and released before we can get there 🙂

**Comment by sorentwo** [comment]
You need to mark the `:oban` dependency as an override:

```elixir
{:oban, github: "oban-bg/oban", override: true},
{:oban_pro, "~> 1.5.0-rc.5", repo: "oban"},
{:oban_web, "~> 2.10", repo: "oban"},
```

**Comment by beerlington** [comment]
Awesome, thanks! I'll open another issue if I have anything to report.
